external-services/opensearch/start.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+if [[ "$HASH_OPENSEARCH_ENABLED" = "true" ]]; then
+  source /usr/share/opensearch/opensearch-docker-entrypoint.sh;
+else
+  echo "Not using Open Search because HASH_OPENSEARCH_ENABLED is not equal to true.";
+  echo "Opensearch is currently disabled by default in Hash to save memory (the opensearch docker image is pretty memory intensive)";
```

Comment:
We are using `Open Search` on line 1 and `Opensearch` on line 2 (like in `index.ts`):

```
  console.log("Opensearch is not enabled. Shutting down search-loader");
```

Let’s pick one ‘official’ style and stick with it.

Reply:
There's also OpenSearch and Opensearch in `packages/hash/README.md` 👀

Reply:
OpenSearch is correct.

Reply:
yeah, I've replaced the ones I could find with OpenSearch after checking the website. There might still be older bits of text in other places that still have it wrong though.